docs/api-opentelemetry.asciidoc

```diff
@@ -7,7 +7,7 @@ endif::[]
 === OpenTelemetry bridge
 
 The Elastic APM OpenTelemetry bridge allows creating Elastic APM `Transactions` and `Spans`,
-using the OpenTelemetry API.
+using the OpenTelemetry API. OpenTelemetry metrics are collected and shipped to elastic as well.
```

Comment:
```suggestion
using the OpenTelemetry API. OpenTelemetry metrics are also collected.
```

`elastic` in itself is a bit vague destination, and we'd better avoid any confusion where the metrics are sent.